resources/views/partials/analytics.blade.php

```diff
@@ -18,3 +18,19 @@
   _gs('{{ $gosquaredTracking }}');
 </script>
 @endif
+@if($piwikTracking = Setting::get('app_analytics_pi_url'))
```

Comment:
You may want to be checking for both `app_analytics_pi_url` and `app_analytics_pi_siteid` to ensure that Piwik is properly configured.